llvm/lib/IR/Verifier.cpp

```diff
@@ -5659,9 +5659,7 @@ void Verifier::visitIntrinsicCall(Intrinsic::ID ID, CallBase &Call) {
     break;
   }
   case Intrinsic::lround:
-  case Intrinsic::llround:
-  case Intrinsic::lrint:
-  case Intrinsic::llrint: {
+  case Intrinsic::llround: {
```

Comment:
Do we need to verify the vector types for lrint/llrint have the same number of elements in the source and destination?